docs/hardware_keyboard_guidelines.md

```diff
@@ -1,18 +1,94 @@
 # QMK Keyboard Guidelines
 
-We welcome all keyboard projects into QMK, but ask that you try to stick to a couple guidelines that help us keep things organised and consistent.
+Since starting QMK has grown by leaps and bounds thanks to people like you who contributed to creating and maintaining our community keyboards. As we have grown we have discovered some patterns that work well, and ask that you conform to them to make it easier for other people to make use of your hard work.
```

Comment:
```suggestion
Since starting, QMK has grown by leaps and bounds thanks to people like you who contributed to creating and maintaining our community keyboards. As we've grown, we've discovered some patterns that work well, and we ask that you conform to them to make it easier for other people to benefit from your hard work.
```
Add punctuation and change wording a bit to reduce repetition.